cmd/serve.go

```diff
@@ -197,8 +206,8 @@ func buildAPIDependencies(
 		userService,
 		projectService)
 
-	sessionService := authenticate.NewSessionManager(postgres.NewSessionRepository(), grpc_interceptors.SessionValidity)
-	registrationService := authenticate.NewRegistrationService(postgres.NewFlowRepository(), userService, cfg.App.Authentication)
+	sessionService := authenticate.NewSessionManager(postgres.NewSessionRepository(dbc), grpc_interceptors.SessionValidity, logger)
```

Comment:
nit: move the logger arg to first argument.

Reply:
Also can you check if logger is part of context already? Using `logger := grpczap.Extract(ctx)`?